src/System.Private.ServiceModel/src/System/ServiceModel/Security/Tokens/SecurityContextSecurityToken.cs

```diff
@@ -12,40 +11,17 @@
 
 namespace System.ServiceModel.Security.Tokens
 {
-    public class SecurityContextSecurityToken : SecurityToken, TimeBoundedCache.IExpirableItem, IDisposable
+    internal class SecurityContextSecurityToken : SecurityToken, TimeBoundedCache.IExpirableItem, IDisposable
```

Comment:
similar question here

Reply:
This type isn't in our contract. We have no internal usage of the public constructors so I removed them. As I was able to also remove most of the unused code in the class, it became clear this class won't be needed to be exposed in our contracts so I moved the class to internal. This is what prompted changing the interface to internal too.